### DISCUSSION TITLE
Flow running dashboard

### DISCUSSION THREAD

**Discussion by Ventinc** [directus/directus#20748] [Feature Requests]
### Summary

Display a flow log and running dashboard, to know if there is a running flow, in which state he is, and potentially more. But for now just a tab with currently running flow and time started and finished with result (Success, Error) could be a cool feature for a lot of project that use flows I think.
Later add granular information for operations within the current running flow with time displayed could also be cool for managing operations optimizations

### Motivation

For long async task, 4-10 minutes or more, it's cool to have a visual feedback on current running tasks

### Detailed Design

It's just a screen from an app I don't know, but something like that: 
![snowsight-task-graph-run](https://github.com/directus/directus/assets/16818589/fa098ea9-0d74-4bb8-a58e-d3056e74844e)

### Must Have:
- Flow task running informations

### Should Have:
- Status (Success or Error)
- Time spend for task (Time started and Time finished)
- What trigger the flow (Event type manual, webhook, etc.), if manual the user who trigger the flow.
- Data input

### Could Have:
- Granular informations for operations (Same as task flow but just a task for operation linked to this task flow)

### Drawbacks

More data in the database. Could make a little slower flows but it's ok, because we need those informations.

### Alternatives

None because currently in the database we do not have informations for running tasks. I could create a model and manage it myself, but I think this feature could benefit everyone from developers to users of directus frontend.

### Adoption Strategy

It's not a breaking change, just an update with a new table `directus_task`. And an update in `executeFlow` and `executeOperation` to add datas in db. A migration should work.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
